keras_cv/models/backbones/resnet_v2/resnet_v2_backbone_test.py

```diff
@@ -75,6 +75,25 @@ def test_saved_model(self, save_format, filename):
         restored_output = restored_model(self.input_batch)
         self.assertAllClose(model_output, restored_output)
 
+    @parameterized.named_parameters(
+        ("tf_format", "tf", "model"),
+        ("keras_format", "keras_v3", "model.keras"),
+    )
+    def test_saved_alias_model(self, save_format, filename):
+        model = ResNet18V2Backbone()
+        model_output = model(self.input_batch)
+        save_path = os.path.join(self.get_temp_dir(), filename)
+        model.save(save_path, save_format=save_format)
+        restored_model = keras.models.load_model(save_path)
+
+        # Check we got the real object back.
+        # Note that these aliases create the base case
```

Comment:
I noticed that `get_registered_name` returns the right class but `serialize_keras_object` does not:
```python
>>> keras.utils.get_registered_name(ResNet18V2Backbone)
'keras_cv.models>ResNet18V2Backbone'
>>> keras.utils.serialize_keras_object(ResNet18V2Backbone())
{'class_name': 'keras_cv.models>ResNetV2Backbone', 'config': {'stackwise_filters': ListWrapper([64, 128, 256, 512]), 'stackwise_blocks': ListWrapper([2, 2, 2, 2]), 'stackwise_strides': ListWrapper([1, 2, 2, 2]), 'include_rescaling': True, 'input_shape': (None, None, 3), 'stackwise_dilations': ListWrapper([1, 1, 1, 1]), 'input_tensor': None, 'block_type': 'basic_block', 'name': 'res_net_v2_backbone_1', 'trainable': True}}
```

Reply:
Upon further analysis this is correct this these args are only in the signature of the base class.